src/Notification.vala

```diff
@@ -0,0 +1,78 @@
+/*
+* Copyright 2020 elementary, Inc. (https://elementary.io)
+*
+* This program is free software; you can redistribute it and/or
+* modify it under the terms of the GNU General Public
+* License as published by the Free Software Foundation; either
+* version 3 of the License, or (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+* General Public License for more details.
+*
+* You should have received a copy of the GNU General Public
+* License along with this program; if not, write to the
+* Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+* Boston, MA 02110-1301 USA
+*
+*/
+
+public class Notifications.Notification : GLib.Object {
+    public GLib.DesktopAppInfo? app_info { get; private set; default = null; }
+    public GLib.NotificationPriority priority { get; private set; default = GLib.NotificationPriority.NORMAL; }
+    public HashTable<string, Variant> hints { get; construct; }
+    public string app_icon { get; construct; }
+    public string app_id { get; private set; default = "gala-other"; }
```

Comment:
`gala-other` was a constant in `DBus.vala` and it looks like it was only used for this. Should probably move it over here.